### DISCUSSION TITLE
Donation link is empty :/

### DISCUSSION THREAD

**Discussion by R1im** [smacke/subsync#168] [General]
Dunno where to post, seems like this is better than under "issues"
Anyway, donation link that refers here https://github.com/krassowski/krassowski/commits/main/README.md
is empty from July

**Comment by smacke** [comment]
Thanks for flagging. That's a shame, it was a good list. Adding back my old donate button in the mean time.

**Reply by R1im** [reply]
I use this to exchange my euros for bullets https://u24.gov.ua

**Comment by smacke** [comment]
Just added an alternative link and a "support Ukraine" badge.
